selfdrive/controls/controlsd.py

```diff
@@ -61,8 +61,8 @@ def __init__(self, sm=None, pm=None, can_sock=None):
       self.can_sock = messaging.sub_sock('can', timeout=can_timeout)
 
     # wait for one health and one CAN packet
-    hw_type = messaging.recv_one(self.sm.sock['health']).health.hwType
-    has_relay = hw_type in [HwType.blackPanda, HwType.uno, HwType.dos]
+    self.hw_type = messaging.recv_one(self.sm.sock['health']).health.hwType
```

Comment:
not necessary, you're already reading the health packet with the other logic

Reply:
Gotcha, will switch to checking the health packet.